corehq/apps/ota/views.py

```diff
@@ -63,8 +63,8 @@ def search(request, domain):
                  .case_type(case_type)
                  .is_closed(False)
                  .size(CASE_SEARCH_MAX_RESULTS))
-    config = CaseSearchConfig(domain=domain).config
-    fuzzies = config.get_fuzzy_properties_for_case_type(case_type)
+    config, _ = CaseSearchConfig.objects.get_or_create(domain=domain)
```

Comment:
How come you chose `get_or_create` over `get` or `get_or_none`? Is there a use case where we would want a request from a device to create a CaseSearchConfig object?